tests/appsec/iast/test_iast.py

```diff
@@ -9,7 +9,7 @@
 @released(
     dotnet="?",
     golang="?",
-    java={"spring-boot": "0.108.0", "*": "?"},
+    java={"spring-boot": "0.108.0", "spring-boot-jetty": "0.109.0", "*": "?"},
```

Comment:
The remain fixes for Java will be added from this PR: https://github.com/DataDog/system-tests/pull/508